integration/helpers.go

```diff
@@ -712,6 +712,49 @@ func (i *TeleInstance) startNode(tconf *service.Config, reverseTunnel bool) (*se
 	return process, nil
 }
 
+func (i *TeleInstance) StartApp(conf *service.Config) (*service.TeleportProcess, error) {
+	dataDir, err := ioutil.TempDir("", "cluster-"+i.Secrets.SiteName)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
```

Comment:
`defer os.RemoveAll(dataDir)`?

Reply:
This directory is supposed to outlive this function iirc. I've dealt with similar cases in my test flakes PR by creating sub-directories in a central temp test-specific directory and removing after the test.

Reply:
I now tack all temporary data directories in an instance and remove them after having stopped them.